src/models/transfers.js

```diff
@@ -376,7 +378,16 @@ function * fulfillTransfer (transferId, fulfillmentUri) {
   }
 }
 
-function * setTransfer (transfer, requestingUser) {
+function * setTransfer (externalTransfer, requestingUser) {
+  const validationResult = validator.create('Transfer')(externalTransfer)
+  if (validationResult.valid !== true) {
+    const message = validationResult.schema
+      ? 'Body did not match schema ' + validationResult.schema
+      : 'Body did not pass validation'
+    throw new InvalidBodyError(message, validationResult.errors)
+  }
+  let transfer = converters.convertToInternalTransfer(externalTransfer)
```

Comment:
Assigned to here: https://github.com/interledger/five-bells-ledger/pull/261/files/2606625fd12d06c70ed07ab40b6be82dfe6e258f#diff-b29546dff0ad4cfb82a7d138568c73f6R423